app/assets/stylesheets/invitations.css.scss

```diff
@@ -5,11 +5,19 @@
 }
```

Comment:
In my experience doing responsive design for a few years now, there is gain in seeking pixel perfection: you have greater control and it ends up looking better. In most cases, I work with 3 basic scenarios/form factors: smartphone, tablet, desktop, and adjust the sizes accordingly. Specially on smartphones, which are a totally different use case.